app/src/components/features/rules/GettingStarted/WorkspaceOnboarding/OnboardingSteps/TeamWorkspaces/WorkspaceCreation.tsx

```diff
@@ -149,7 +157,10 @@ export const CreateWorkspace: React.FC<Props> = ({ defaultTeamData }) => {
           //@ts-ignore
           type="email"
           value={inviteEmails}
-          onChange={setInviteEmails}
+          onChange={(email) => {
+            setInviteEmails(email);
+            trackTeamMemberAdded("onboarding");
```

Comment:
Shouldn't it be `trackAddMemberClicked` and memberAdded should be when the members have been added to the team?